Python/ceval.c

```diff
@@ -13,7 +13,7 @@
 #include "pycore_object.h"        // _PyObject_GC_TRACK()
 #include "pycore_moduleobject.h"  // PyModuleObject
 #include "pycore_opcode.h"        // EXTRA_CASES
-#include "pycore_pyerrors.h"      // _PyErr_Fetch()
+#include "pycore_pyerrors.h"      // _PyErr_Fetch(), _PyErr_GetRaisedException()
```

Comment:
Remove the `_PyErr_Fetch` from the comment?

Reply:
There still are a couple left in the tracing stuff (the trace function wants the triplet).